templates/login.html

```diff
@@ -20,7 +20,7 @@ <h1>Microsoft Identity Python Web App</h1>
     {% endif %}
 
     {% if config.get("B2C_RESET_PASSWORD_AUTHORITY") %}
-    <a href="{{config.get('B2C_RESET_PASSWORD_AUTHORITY')}}?client_id={{config.get('CLIENT_ID')}}">Reset Password</a>
+    <a href='{{Auth(session={}, authority=config["B2C_RESET_PASSWORD_AUTHORITY"], client_id=config["CLIENT_ID"]).log_in(redirect_uri=url_for("auth_response", _external=True))["auth_uri"]}}'>Reset Password</a>
```

Comment:
@pamelafox , W.R.T. #108, currently I choose to keep the reset password link here, partially because it gives maintainers an easy way to test this PR. For example, I was able to test this behavior using your "pythonb2c" test tenant and test app even when I do not have your test app's secret. :-)

Reply:
This is a lot of code to put inside a Jinja expression. It's valid Jinja, but it seems error-prone, what do you think of passing it in from the Python code instead?

I'm fine with the reset password link if it works, it didn't before, but I see you've fixed the link template to be functional.

Reply:
> This is a lot of code to put inside a Jinja expression. It's valid Jinja, but it seems error-prone, what do you think of passing it in from the Python code instead?

That is an accurate and fair observation. We were aware of that and still chose to do it this way since day 1 (of introducing the B2C into this sample), because a Reset Password (or Edit Profile) link conceptually belongs to an html template. The coincidence of them being lengthy and error-prone is an unfortunate side effect of B2C's design. As a trade-off, we believe the current approach is good enough in that the link is correct in our sample and then our customers will not need to modify it at all.